search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -110,8 +110,7 @@
     :always-case-sensitive #{}
     :disallow-pattern #{}}))
 
-;; TODO: Generic work: multiple values need to be pulled from config files
```

Comment:
Should we keep the TODO?

Reply:
We do want to implement it, but we promised to get rid of these comments and write tickets a couple of PIs ago, So I am doing that when I see some leftovers.